src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramBuilder.cpp

```diff
@@ -1302,7 +1306,7 @@ class ShaderFragmentHeaderReadTex : public ShaderPart
 		if (!m_glinfo.isGLES2) {
 
 			if (g_textureConvert.useTextureFiltering()) {
-				shaderPart += "uniform lowp int uTextureFilterMode;								\n";
+				//shaderPart += "uniform lowp int uTextureFilterMode;								\n";
```

Comment:
Just remove it.

Reply:
I restored it to its original place. I had moved it because I was using it elsewhere at some point, but I am no longer using it.